libs/deposits/src/lib/deposit-limits.tsx

```diff
@@ -7,6 +7,10 @@ interface DepositLimitsProps {
   balance?: BigNumber;
 }
 
+const format = (value: string) => {
```

Comment:
chore (blocking): If we need this function, then there is a problem elsewhere and its worth fixing. We shouldn't ever have NaN here.